collections/_article/gdc-2024-retrospective.md

```diff
@@ -0,0 +1,35 @@
+
+![Exhibitors at the Godot booth](/storage/blog/gdc/2024/booth.png)
+
+Of course, you cannot attend a big industry gathering without scheduling a fair share of meetings, so the Foundation did just that. One goal was to secure funding to keep the project running and be able to hire more core developers. Another recurring topic in such conversations were possible integrations and better industry support for Godot developers - we cannot wait to show you what is brewing in that area.
+
+__If you or your company had packed schedules already or could not attend GDC, but want to talk to us about similar topics, please reach out to [contact@godotengine.org](mailto:contact@godotengine.org)__
+
+Our gathering at GitHub HQ also proved successful, with many Godot users, content creators, and contributors chatting about their personal projects with us. The vegan food seemed to be a hit as well. Afterwards we were told it was one of the most relaxed evenings of the week for some of our guests. We are grateful to GitHub that we could provide this kind of space to the Godot community, and are looking forward to GodotCon in that regard (TBA).
+
+![Assorted pictures from the week](/storage/blog/gdc/2024/collage.png)
+
+As for the team itself, this conference allowed the fully remote team to assemble in person again. Between booth duty, team meetings, and extracurricular activities, we are happy to report we all grew a little closer. Even engine developers need to leave their houses sometimes ;)
+
+At the end of the day, we and everyone we met are passionate about this open source project  — interacting with the community in person is a really helpful tool to remind everyone of that common ground, and use it to our advantage: to compare feedback and sentiment to what we hear on the internet, and to forge lasting relationships. Which is one of the key reasons why we are now looking into being present at more conferences across the world in the future, aiming to enable more of these important conversations and super helpful learnings. This will come at the expense of a smaller GDC booth next year however, as we have to split budget and personpower accordingly. San Francisco indeed proved as pricey as everybody keeps saying, and we strongly believe there is more to gain from diversifying opposite to always being in one and the same location. 
+
+Did you visit the booth? Snapped any pictures you want to share with us? Use the forum post [linked here]() to tell us about your experience at GDC, we would love to hear back from you :)
```

Comment:
Don't forget to add the link here :slightly_smiling_face:

Reply:
Can only do so when the blog post is live (since the forum post contains a link to the article), but I will keep it in mind!